packages/smooth_app/lib/pages/preferences/user_preferences_connect.dart

```diff
@@ -119,18 +122,19 @@ class UserPreferencesConnect extends AbstractUserPreferences {
         ),
         _getListTile(
           title: appLocalizations.contact_title_press_email,
-          leadingIconData: Icons.drafts_outlined,
+          leadingIconData: Icons.drafts,
           onTap: () async => _sendEmail(
             recipient:
                 ProductQuery.getLanguage() == OpenFoodFactsLanguage.FRENCH
                     ? 'presse@openfoodfacts.org'
                     : 'press@openfoodfacts.org',
           ),
         ),
+        const Divider(),
         _getListTile(
           title: appLocalizations.contact_title_newsletter,
           url: 'https://link.openfoodfacts.org/newsletter-en',
-          leadingIconData: Icons.subscriptions_outlined,
+          leadingIconData: CupertinoIcons.news_solid,
```

Comment:
Can you also use it for contact_title_press_page ?

Reply:
> Can you also use it for contact_title_press_page ?

Done